### ISSUE TITLE
Quick Toggle Filters

### ISSUE THREAD

**Issue by jk3us** [serge-rider/dbeaver#1074]
Right now, there is an option in the context menu for schemas, tables, etc, an option to clear the filter.  I think it would be much more helpful to be able to toggle the filter (like you can with the checkbox in the dialog box), so that you can quickly re-enable it later.  If I have a big filter set up, sometimes I need to turn it off for a minute to get to something I normally don't want to see, but I then want to turn the filter back on.  If I could do that without having to open the filter dialog, that would be nice.

![image](https://cloud.githubusercontent.com/assets/41621/21276590/b61f1608-c397-11e6-96e8-bbc04b202aa1.png)

In other words, "Clear Filter" should be "Disable (or Enable) Filter".

**Comment by serge-rider** [comment]
Filter toggle command will be added in the next version. 
No keyboard shortcut is defined by default but you can assign one in preferences.
